### ISSUE TITLE
Sybil Report

### ISSUE THREAD

**Issue by devstude** [connext/community-sybil-reports#252]
# Related Addresses
0x623fe782c26a810539cf53c905c06dff44010aec

# Reasoning

All addresses where initially funded on Ethereum mainnet from same main account 0x623fe782c26a810539cf53c905c06dff44010aec (which is also eligible for the Connext user airdrop) in 4 batches:
1st batch of funding 4 wallets for the same amount of 0.02 eth that was made 2021-12-30 in a same second
0x623fe782c26a810539cf53c905c06dff44010aec
0x623fe782c26a810539cf53c905c06dff44010aec
0x623fe782c26a810539cf53c905c06dff44010aec
0x623fe782c26a810539cf53c905c06dff44010aec

2nd batch same date as first 2021-12-30 but few hours later. Sybil initially funded another 10 wallets with 0.02-0.023 eth. All transactions were made in 2 seconds.
0xcae91aad622d11a6ecb20891235c65e8017d976e
0x1984e575b783e094a68ff775bd866fec96f7d564
0x055e5bb2789b807e755613b4c09c9ee576ad6f1e
0xe7ed7d4007193e0304541cfee9c7324fc8fec460
0xa12f1c61910f91612e849466f734aa6d1a8aebbd
0x196cfb78aa6156998555cc7c39940250e8e180ac
0xb58ec0ed19054680f16c9a7dd57aca4d9029248a
0xfba794ce471615d52cb5dddefb217ac8d77a6734
0xfb8ad8896b491ca71e1b38f0b814393a714fd01d
0x080a6c46bb8cf4b82e0077fe601bc894d8512477

next day 2021-12-31 he funded another 3 wallets (in a same second) and amount 0.02175-0.023 eth
0x75fd8a28c9ed1f6eac77da005cfc47fcd2f81186
0xc9ace8782f457c10077f82ac2075a9219abe128b
0x0070ba45e5b5c442734b755cfe3cd2c1c1074ce3

and half an hour later another batch of fundings 16 new wallets for 0.02175-0.023 eth (all in same second)
0x5027b3e69d3c088cea2ad70f0ac87631cf6d87f9
0x78b99884a99337fb2883338516159d3e12ff2f05
0x9d87b5c99dbcfb65ae58ee2f5fc0813967155324
0x829864a0a1b439f58d740128348aaea745ed88a3
0xa72b6cbb345eb156c981fc0cbbb53e96305f756a
0x6d31fbf91fbabf8a7272fe8629f75e4c664dd048
0x3d020b5b71221e66f179acecc82bf2a089acc90d
0x8ca9d81ced56ab721d599270a8601d2db757e30f
0x064dd52192ba2e646cb8d7ef6f93beff1d4a10bc
0x4f2bf3233a3e667251d6e7f41ab4a9565014cc62
0xf0e4ed643d99c0f93b5bcde280fac7c8494a01f4
0x0c462cc3fe3c8ebe5ccc09824d8963a24b67fdd0
0xebfa1310785253e286aad091b015e29cd54bf990
0xc17deebf6d8f87a6119abca872ff3f658fd1f5a5
0x29cc00445c548ed858abcbdf8aa8449a121a0ad0
0x165c690a264665a253ffe3dedbed6938a3c066bf

and 8 minutes later he finally funded same way this wallet
0x4f37f991cdc3dab2570be339eed1af01659780a5

All of those accounts are eligible for Connext user airdrop!

# Methodology

Those accounts where found by running dune scripts that analyze initial wallet fundings (for wallets eligible for the Connext user airdrop) and manual checking of same time/value/initial funder wallets. 

# Rewards Address

0xEd3aAD546d663084794CcCcA6987089395FC1760

**Comment by devstude** [comment]
Oh, I missclicked. First block of 4 wallets should to contain those:

0x775454c8f352a9654eeac732e968173350044560
0x030d40a52eee98f6e349498c5499347422bad754
0x447075280ac9d8297483582d0c32647ba7f9b0e1
0xd35a157325e9537cdc1c660662f8c6ddef19ce96

**Comment by devstude** [comment]
I fucked up a Related Addresses section of original post. But it is easy to see a list of sybil addresses I found down by the text of Issue. They are:
0x775454c8f352a9654eeac732e968173350044560
0x030d40a52eee98f6e349498c5499347422bad754
0x447075280ac9d8297483582d0c32647ba7f9b0e1
0xd35a157325e9537cdc1c660662f8c6ddef19ce96
0xcae91aad622d11a6ecb20891235c65e8017d976e
0x1984e575b783e094a68ff775bd866fec96f7d564
0x055e5bb2789b807e755613b4c09c9ee576ad6f1e
0xe7ed7d4007193e0304541cfee9c7324fc8fec460
0xa12f1c61910f91612e849466f734aa6d1a8aebbd
0x196cfb78aa6156998555cc7c39940250e8e180ac
0xb58ec0ed19054680f16c9a7dd57aca4d9029248a
0xfba794ce471615d52cb5dddefb217ac8d77a6734
0xfb8ad8896b491ca71e1b38f0b814393a714fd01d
0x080a6c46bb8cf4b82e0077fe601bc894d8512477
0x75fd8a28c9ed1f6eac77da005cfc47fcd2f81186
0xc9ace8782f457c10077f82ac2075a9219abe128b
0x0070ba45e5b5c442734b755cfe3cd2c1c1074ce3
0x5027b3e69d3c088cea2ad70f0ac87631cf6d87f9
0x78b99884a99337fb2883338516159d3e12ff2f05
0x9d87b5c99dbcfb65ae58ee2f5fc0813967155324
0x829864a0a1b439f58d740128348aaea745ed88a3
0xa72b6cbb345eb156c981fc0cbbb53e96305f756a
0x6d31fbf91fbabf8a7272fe8629f75e4c664dd048
0x3d020b5b71221e66f179acecc82bf2a089acc90d
0x8ca9d81ced56ab721d599270a8601d2db757e30f
0x064dd52192ba2e646cb8d7ef6f93beff1d4a10bc
0x4f2bf3233a3e667251d6e7f41ab4a9565014cc62
0xf0e4ed643d99c0f93b5bcde280fac7c8494a01f4
0x0c462cc3fe3c8ebe5ccc09824d8963a24b67fdd0
0xebfa1310785253e286aad091b015e29cd54bf990
0xc17deebf6d8f87a6119abca872ff3f658fd1f5a5
0x29cc00445c548ed858abcbdf8aa8449a121a0ad0
0x165c690a264665a253ffe3dedbed6938a3c066bf
0x4f37f991cdc3dab2570be339eed1af01659780a5

On a next day they have been reported in Issue #403 by a guy who trying to steal my rewards for work I done.
